AsyncDisplayKit/Layout/ASDimension.h

```diff
@@ -191,6 +191,15 @@ ASDISPLAYNODE_INLINE AS_WARN_UNUSED_RESULT NSString *NSStringFromASLayoutSize(AS
           NSStringFromASDimension(size.height)];
 }
 
+#pragma mark - ASEdgeInsets
+
+typedef struct {
+  ASDimension top;
+  ASDimension left;
+  ASDimension bottom;
+  ASDimension right;
+} ASEdgeInsets;
```

Comment:
I think this type could be generally useful for many things, so proposing we keep it in ASDimension.  I can move it if needed.